PyQt5-stubs/sip.pyi

```diff
@@ -63,7 +63,7 @@ class array(Sequence[_T], Generic[_T]):
 # The voidptr type.
 class voidptr:
 
-    def __init__(addr: Union[int, Buffer], size: int = -1, writeable: bool = True) -> None: ...
+    def __init__(addr: Union[int, None, bytes, bytearray, 'voidptr'], size: int = -1, writeable: bool = True) -> None: ...
```

Comment:
Is this `__init__` somehow static?

Reply:
Heh, that's fun.  Let's see how this works out.

```suggestion
    def __init__(self, addr: Union[int, None, bytes, bytearray, 'voidptr'], size: int = -1, writeable: bool = True) -> None: ...
```